notebooks/experimental/semantic_operators.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Nit: Avoid "we". Also, avoid words like "small". If it is difficult for the customer to use and you say it's "small" or "easy" it will make their frustration worse.</p><p> </p><p>I suggest:</p><p> </p><p>Learn about each semantic operator by trying some examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/googleapis/python-bigquery-dataframes/pull/1243/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>